### PR TITLE
relocate lead inspector assignment logic

### PR DESCRIPTION
**Pull request by mmumby**
- moved lead inspector assignment logic to the technical review tab
Screen Shot 2020-11-03 at 3 05 03 PM

- if the application is imported, hide "Verification"
- Updated name of "technical review" to "application" 
- Removed lead inspector modal, added ability to update on the UI of admin tab
Screen Shot 2020-11-03 at 3 04 10 PM

### CODE REVIEW

**Review by nayr974 — COMMENTED** [review]
Looks good, but one question

File: services/core-web/src/components/noticeOfWork/applications/NOWSideMenu.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
[the remark this answers is not in the document]
Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/NOWSideMenu.js]: ooops, removed!

File: services/core-web/src/components/Forms/noticeOfWork/UpdateNOWLeadInspectorForm.js [hunk 2]
```diff
@@ -1,34 +1,70 @@
...
             name="lead_inspector_party_guid"
-            label="Lead Inspector"
+            label={
+              !props.isAdminView
+                ? "Assign a lead inspector before continuing. This assignment can be updated later under the Administrative tab."
+                : ""
+            }
             component={renderConfig.GROUPED_SELECT}
-            placeholder="Start typing lead inspector name"
+            placeholder="Start typing lead inspectors name"
```
Comment (exzizt) [inline services/core-web/src/components/Forms/noticeOfWork/UpdateNOWLeadInspectorForm.js:39]: Should be "lead inspector's name", but this still sounds weird without an article like "a" or "the"...
What about "Enter Lead Inspector Name" or "Start typing the lead inspector's name"?

**Review by exzizt — APPROVED** [review]
Looks good, just one comment about the English language 😝

**Pull request merged**